content/en/docs/collector/deployment/gateway/index.md

```diff
@@ -87,29 +86,28 @@ upstream collector4318 {
 }
 ```
 
-### load-balancing exporter
+### Load-balancing exporter
 
-For a concrete example of the centralized collector deployment pattern we first
-need to have a closer look at the load-balancing exporter. It has two main
+For a concrete example of the centralized Collector deployment pattern, first
+look at the load-balancing exporter. It has two main
 configuration fields:
 
-- The `resolver`, which determines where to find the downstream collectors (or:
-  backends). If you use the `static` sub-key here, you will have to manually
-  enumerate the collector URLs. The other supported resolver is the DNS resolver
-  which will periodically check for updates and resolve IP addresses. For this
-  resolver type, the `hostname` sub-key specifies the hostname to query in order
-  to obtain the list of IP addresses.
-- With the `routing_key` field you tell the load-balancing exporter to route
-  spans to specific downstream collectors. If you set this field to `traceID`
-  (default) then the Load-balancing exporter exports spans based on their
-  `traceID`. Otherwise, if you use `service` as the value for `routing_key`, it
-  exports spans based on their service name which is useful when using
-  connectors like the [Span Metrics connector][spanmetrics-connector], so all
-  spans of a service will be sent to the same downstream collector for metric
+- The `resolver`, which determines where to find the downstream Collectors (or
+  backends). If you use the `static` subkey here, you must manually
+  enumerate the Collector URLs. The other supported resolver is the DNS resolver,
+  which periodically checks for updates and resolves IP addresses. For this
+  resolver type, the `hostname` subkey specifies the hostname to query to
+  obtain the list of IP addresses.
+- Use the `routing_key` field to route
+  spans to specific downstream Collectors. If you set this field to `traceID`
+  (default), the load-balancing exporter exports spans based on their
```

Comment:
I'm not sure that `traceID` is still the default value for `routing_key`. At least, I couldn't find confirmation in the README. @open-telemetry/collector-contrib-approvers, could anyone confirm one way or the other?
 
```suggestion
- The `routing_key` field routes
  spans to specific downstream Collectors. If you set this field to `traceID`, the load-balancing exporter exports spans based on their
```